djtriggers/tests/test_logic.py

```diff
@@ -0,0 +1,60 @@
+from datetime import timedelta
+
+import mock
+from django.test import override_settings
+from django.test.testcases import TestCase
+from django.utils import timezone
+
+from djtriggers.logic import process_triggers
+from djtriggers.tests.factories.triggers import NormalDummyTriggerFactory, FailingDummyTriggerFactory
+
+
+class LogicTest(TestCase):
+
+    @staticmethod
```

Comment:
You define it as static, but then later on use it as an instance method. Then I would just remove the decorator and be done with it.